### PR TITLE
docs(readme): update the release section

### PR DESCRIPTION
**Pull request by Linkid**
Need the project lead validation, to be sure ;).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,8 +35,12 @@ If you have great ideas for Auditlog, or if you like to improve something, feel
...
-2. Create a new branch `vX.Y.Z` from master for that specific release.
-3. Bump versions in `setup.py` and `docs/source/conf.py` (docs have 2 places where the versions need to be changed!)
-4. Pull request `vX.Y.Z` -> `master`. 
-5. Pull request `master` -> `stable`. This merge triggers the deploy to pypi.
+1. Make sure all tests on `master` are green
+2. Create a new branch `vX.Y.Z` from master for that specific release
+3. Update the CHANGELOG release date
+4. Pull request `vX.Y.Z` -> `master`
+5. As a project lead, once the PR is merged, create and push a tag `vX.Y.Z`: this will trigger the release build and a notification will be sent from Jazzband of the availability of two packages (tgz and wheel)
+6. Test the install
```
Comment (hramezani) [inline README.md:43]: What does this step mean?

File: README.md [hunk 2]
```diff
@@ -35,8 +35,12 @@ If you have great ideas for Auditlog, or if you like to improve something, feel
...
+7. Publish the release to PyPI
+8. Pull request `master` -> `stable`
```
Comment (hramezani) [inline README.md:45]: Do we really need to do this?

File: README.md (hunk 1 above)
Comment (Linkid) [reply README.md:43]: Maybe I should be more explicit, yes ^^. One should install the generated wheel and maybe run tests to be sure everything is working as expected.

File: README.md (hunk 2 above)
Comment (Linkid) [reply README.md:45]: It is always a good thing to keep a branch with the latest major / minor version to apply patches. Maybe the name is not adapted to our needs. What do you think about that process ? It is the same as Django.

Comment (hramezani) [reply README.md:45]: I think we don't need this in this project. because we only maintain one version of the package.

Comment (hramezani) [inline README.md:45]:
```suggestion
```

File: README.md [hunk 3]
```diff
@@ -35,8 +35,12 @@ If you have great ideas for Auditlog, or if you like to improve something, feel
...
+9. Update the CHANGELOG with the future version
```
Comment (hramezani) [inline README.md:46]:
```suggestion
```

**Review by hramezani — APPROVED** [review]

**Pull request merged**